modules/core/classes/Gallery.class

```diff
@@ -167,6 +167,9 @@ class Gallery {
      */
     var $_phpVm = null;
 
+    /* deprecated dynamic properties in php 8.2 */
+    var $_squareQueue;
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Without the variable declaration unit tests 1935 and 1938 through 1940 fails with the deprecation warning from php 8.2